### PR TITLE
Post process

### PR DESCRIPTION
**Pull request by MatheMax**
I am still working on this issue. 
Please do not check it right now.

### CODE REVIEW

**Comment by MatheMax** [comment]
Hey @kkmann 
I finished my work on this PR.
However, travis is not happy with me, but on my local computer, the devtools check does not complain.
Could you maybe restart the Travis check?

**Comment by kkmann** [comment]
Seems to be a test issue, anything randomized without a seed? 

https://travis-ci.org/kkmann/otsd/builds/482494089?utm_source=github_status&utm_medium=notification

File: R/GroupSequentialDesign.R [hunk 1]
```diff
@@ -35,54 +35,28 @@ GSDesign <- function(n1, c1f, c1e, n2_pivots, c2_pivots, x1_norm_pivots, weights
...
     if (any(weights <= 0))
         stop("weights must be positive")
-    new("GSDesign", n1 = n1, c1f = c1f, c1e = c1e, n2_pivots = n2_pivots,
-        c2_pivots = c2_pivots, x1_norm_pivots = x1_norm_pivots, weights = weights)
+    tunable <- logical(8) # initialize to all false
+    tunable[1:5] <- TRUE
+    names(tunable) <- c("n1", "c1f", "c1e", "n2_pivots", "c2_pivots", "x1_norm_pivots", "weights", "tunable")
+        new("GSDesign", n1 = n1, c1f = c1f, c1e = c1e, n2_pivots = n2_pivots,
+        c2_pivots = c2_pivots, x1_norm_pivots = x1_norm_pivots, weights = weights,
+        tunable = tunable, rounded = FALSE)
```
Comment (kkmann) [inline R/GroupSequentialDesign.R]: I get the point of introducing a new filed 'rounded' - but don't you think this a bit over the top? I'd just implemented it as n2(x1, round = FALSE) in the corresponding method - keeping changes and complexity low.
If you want to keep it, I am fine with it but it is a bit over-complicated ... (if I am not missing something)

Ah, okay - maybe I see. Making this a design property means we do not need to alter anything in the score evaluation? Good point

File: R/minimize.R [hunk 2]
```diff
@@ -49,8 +51,65 @@ minimize <- function(objective, subject_to, initial_design,
...
+
+            return(cont_design)
+
+
+        } else{
 
         return(update(initial_design, res$solution))
+        }
+
+        # TODO: error handling
```
Comment (kkmann) [inline R/minimize.R]: well, we can delete that if we are not doing it ;)

**Comment by kkmann** [comment]
hm, still same error - any idea?

**Comment by MatheMax** [comment]
Okay my idea did not work, I'll try it tomorrow ;)

**Comment by kkmann** [comment]
don't get it - looks like a deterministic error. looks like the error is already in the first commit - they all failed

**Comment by MatheMax** [comment]
I think I found the error.
When we use a quite small effect and a Continuous Prior, then we have to update the support of the prior in order to still obtain sensitive integrals. Otherwise, the support can be too large for correct integration. 
I try to implement this

File: tests/testthat/test_post_processing.R [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        c1f = 0,
+        c1e = 2,
+        n2 = 10 * log(seq(600.0, 100.0, length.out = order)), #use something not too smooth
+        c2 = rep( 1.96, order),
+        order = order)
+
+    # Try a continuous prior
+    null        <- PointMassPrior(.0, 1)
+    dichte_alt  <- function(x) dnorm(x, mean = .3, sd = .1)
+    alternative <- ContinuousPrior(dichte_alt, c(-5, 5))
```
Comment (kkmann) [inline tests/testthat/test_post_processing.R]: why not use ContinuousPrior(dichte_alt, c(-1, 2)) ?

**Comment by kkmann** [comment]
why not use ContinuousPrior(dichte_alt, c(-1, 2)) ?

File: tests/testthat/test_post_processing.R (hunk 3 above)
Comment (MatheMax) [reply tests/testthat/test_post_processing.R]: This is a solution for this specific problem
But I think with another prior there could be again problems and it should not be the goal that the user has to look for the fitting support for every new prior, see my comment above

**Comment by MatheMax** [comment]
Hey @kkmann 
I uploaded a version that should hopefully (let's wait for travis) work.
However, it is not a stable version because it strongly depends on the support of the prior. 
I open a new issue for this topic.

**Pull request merged**